src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1211,7 +1211,7 @@ static ErrorCode getRefMismatchErrorCode(TypeKind type)
             return null;
         }
 
-        private ConstantValue FoldConstantNumericConversion(
+        private ConstantValue? FoldConstantNumericConversion(
```

Comment:
>FoldConstantNumericConversion [](start = 31, length = 29)

Are all existing consumers prepared able to deal with null as the result?

Reply:
Yes, although I've added a few more annotations and handled `null` in one case that is currently obscured by [https://github.com/dotnet/roslyn/issues/45625](https://github.com/dotnet/roslyn/issues/45625).

---
In reply to: [449108702](https://github.com/dotnet/roslyn/pull/45581#discussion_r449108702) [](ancestors = 449108702)